framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/template/TemplateSelector.java

```diff
@@ -0,0 +1,36 @@
+package org.kie.cloud.openshift.template;
+
+import java.util.Arrays;
+
+class TemplateSelector {
+
+    enum Database {
+        postgresql,
```

Comment:
Maybe I'm wrong, but doesn't the code style dictate that enum values are all upper-case?

Reply:
Changed the names to all uppercase. I just wanted to avoid case conversion when comparing with potential string values. But in the end I guess that `equalsIgnoringCase` doesn't hurt..